buildrelease.sh

```diff
@@ -48,13 +48,31 @@ done
 cd docs
 ./builddocs.sh root $projects
 
-echo "Release build complete for the following projects:"
+cd ..
+rm -rf releasedocs
+git clone https://github.com/GoogleCloudPlatform/google-cloud-dotnet.git releasedocs -c core.autocrlf=input
+cd releasedocs
+git checkout gh-pages
+
+for project in $projects
+do
+  rm -r ./docs/$project
+  mv ../docs/output/assembled/$project ./docs
+done
+
+rm -r ./index.html
```

Comment:
Why the -r when this is a file rather than a directory?

Reply:
No reason, fixed.